ocdata/databases/update.py

```diff
@@ -1,3 +1,9 @@
+"""
+Script for updating ase pkl and db files from v3.19 to v3.21.
+Run it with ase v3.19.
```

Comment:
We should assert somewhere in the core scripts that the correct version of ase is being used